container/go/cmd/pusher/pusher.go

```diff
@@ -181,60 +150,14 @@ func push(dst string, img v1.Image) error {
 		}
 	}
 
-	options := []remote.Option{remote.WithAuthFromKeychain(authn.DefaultKeychain)}
-
-	configPath := path.Join(os.Getenv("DOCKER_CONFIG"), "config.json")
-	if _, err := os.Stat(configPath); err == nil {
-		file, err := os.Open(configPath)
-		if err != nil {
-			return errors.Wrapf(err, "unable to open docker config")
-		}
-
-		var dockerConfig dockerHeaders
-		err = json.NewDecoder(file).Decode(&dockerConfig)
-		if err != nil {
-			return errors.Wrapf(err, "error parsing docker config")
-		}
-
-		httpTransportOption := remote.WithTransport(&headerTransport{
-			inner:       newTransport(),
-			httpHeaders: dockerConfig.HTTPHeaders,
-		})
-
-		options = append(options, httpTransportOption)
+	remoteOptions, err := utils.ComputeRemoteWriteOptions(context.Background(), "")
+	if err != nil {
+		return errors.Wrap(err, "unable to compute remote options")
```

Comment:
We should really remove all usage of `errors.Wrap`.

Reply:
Yes, but rather in another pull request, to migrate all the Go code of this repository to Go errors 1.13